python/src/nanoarrow/schema.py

```diff
@@ -214,18 +255,18 @@ def scale(self) -> int:
         return self._c_schema_view.decimal_scale
 
     @property
-    def n_children(self) -> int:
+    def n_fields(self) -> int:
```

Comment:
Is there a specific reason you switched from "children" to "fields"? (the spec speaks about "children", I think?)